.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
-# Eliza should review everything unassigned.
-*                               @hawkw @tokio-rs/tracing
+# Eliza should review everything unassigned, with David as a backup.
+*                               @hawkw @davidbarsky @tokio-rs/tracing
```

Comment:
fwiw, you're already a member of the @tokio-rs/tracing group, so you should get tagged already regardless.